samples/apps/bookinfo/rules/destination-policy-reviews.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: config.istio.io/v1beta1
+kind: DestinationPolicy
+metadata:
+  name: reviews-random
+spec:
+  destination:
+    name: reviews
+  loadBalancing:
+    name: RANDOM
```

Comment:
May be change to round robin? Our default lb is random

Reply:
Again, we're not changing the test since it's super-flaky. Let's just this to pass.